ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/winc/deprovision/cucushift-installer-rehearse-aws-ipi-winc-deprovision-chain.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/winc/deprovision/cucushift-installer-rehearse-aws-ipi-winc-deprovision-chain.yaml",
+	"owners": {
+		"approvers": [
+			"jianlinliu",
+			"yunjiang29",
+			"gpei"
+		],
+		"reviewers": [
+			"jianlinliu",
+			"yunjiang29",
+			"gpei",
+			"jfrancoa",
+			"rrasouli"
+		]
+	}
+}
```

Comment:
New line

Reply:
You can check other metadata files from the repo, like https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/kubevirt/provision/cucushift-installer-rehearse-aws-ipi-ovn-kubevirt-provision-chain.metadata.json, for example, and none of them have extra new line.